src/ui/components/shared/Dialog.tsx

```diff
@@ -4,7 +4,7 @@ import ReplayLogo from "./ReplayLogo";
 
 export type DialogPropTypes = HTMLProps<HTMLDivElement>;
 
-export function Dialog({ children, className, ...props }: DialogPropTypes) {
+export function Dialog({ children, className, showFooterLinks, ...props }: DialogPropTypes) {
```

Comment:
I suspect that you will need to add this to the type declaration on line 5 in order to make the typecheck pass.